catalog/transformer/catalog-transformer-rtf/src/test/java/ddf/catalog/transformer/output/rtf/BaseTestConfiguration.java

```diff
@@ -40,22 +41,23 @@
 
   static final String EMPTY_ATTRIBUTE = "Empty";
   static final String SIMPLE_ATTRIBUTE = "Simple";
+  static final String EXTENDED_ATTRIBUTE = "ext.extended-attribute";
 
   private static final String UNKNOWN_ATTRIBUTE = "Unknown";
 
   public List<RtfCategory> getCategories() {
-    return Arrays.asList(
+    return Stream.of(
```

Comment:
:question: Why recreate a stream and a list every time here instead of declaring it `static`?